great_expectations_cloud/agent/agent.py

```diff
@@ -246,29 +246,27 @@ def _handle_event(self, event_context: EventContext) -> ActionResult:
         """
         # warning:  this method will not be executed in the main thread
 
-        data_context = self.get_data_context(event_context=event_context)
```

Comment:
Trying to reconcile this change with needing to have a DataContext-per-Event in the Runner. How does `get_data_context()` handle it?

Reply:
just read the description more closely :) makes sense

Reply:
Awesome! Yeah from the looks of it, the Runner is indeed using the event_context on each call to `get_data_context`, but the Agent isn't. Python also doesn't really care about overridden class methods having matching function signatures, so it _should_ be safe to delete. I'll run some tests to make sure though.